elide-async/src/test/java/com/yahoo/elide/async/export/TableExporterTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.async.export;
+
+import static com.yahoo.elide.core.EntityDictionary.NO_VERSION;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.when;
+
+import com.yahoo.elide.Elide;
+import com.yahoo.elide.ElideSettings;
+import com.yahoo.elide.async.models.AsyncQuery;
+import com.yahoo.elide.async.models.QueryType;
+import com.yahoo.elide.audit.AuditLogger;
+import com.yahoo.elide.core.DataStore;
+import com.yahoo.elide.core.DataStoreTransaction;
+import com.yahoo.elide.core.EntityDictionary;
+import com.yahoo.elide.core.PersistentResource;
+import com.yahoo.elide.core.TransactionRegistry;
+import com.yahoo.elide.request.EntityProjection;
+import com.yahoo.elide.security.User;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import io.reactivex.Observable;
+
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.UUID;
+
+public class TableExporterTest {
+
+    private DataStoreTransaction tx = mock(DataStoreTransaction.class);
+    private Elide elide;
+    private User user;
+    private GraphQLParser graphQLParser;
+    private AsyncQuery asyncQuery;
+
+    @BeforeEach
+    public void beforeTest() {
+        reset(tx);
+        elide = mock(Elide.class);
+        user = mock(User.class);
+        asyncQuery = mock(AsyncQuery.class);
+        graphQLParser = mock(GraphQLParser.class);
+
+        TransactionRegistry transactionRegistry = mock(TransactionRegistry.class);
```

Comment:
I feel like mocking all of these objects make the tests brittle.  Also, most mocks are likely not global and should be created per test.  I would focus on mocking only the challenging objects (like the data store or something).  Everything else should use real objects.